db.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
+import pandas as pd
+
+
+class DB:
+    def __init__(self, dbname="./Database/save.txt"):
```

Comment:
/Database/save.txt file extention for sqlite should be different not `.txt`

Reply:
Done